code/game/objects/effects/decals/remains.dm

```diff
@@ -28,6 +28,13 @@
 	icon_state = "remainsrobot"
 	anchored = TRUE
 
+/obj/effect/decal/remains/robot/decompile_act(obj/item/matter_decompiler/C, mob/user)
+	C.stored_comms["glass"] += 1
+	C.stored_comms["metal"] += 2
+	C.stored_comms["plastic"] += 2
```

Comment:
I would argue that you should remove plastic from the decompiler, as drones and engineer borgs can't actually use plastic right now.

Reply:
Many items are only plastic, however. How would you correct that?

Reply:
I would probably just delete it and then return. The decompiler can still act as a cleaning tool for those items, but it just wont give anything back.

Reply:
I made them give metal or glass depending on what it was. Grind up a lizard and melt it into glass for example.